spec.bs

```diff
@@ -626,6 +686,9 @@ members:
     :: Any [=WebAuthn extensions=] that should be used for the passed
         credential(s). The caller does not need to specify the
         [[#sctn-payment-extension-registration| payment extension]]; it is added
+        automatically. When setting a non-default
+        {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}
+        then {{AuthenticationExtensionsPaymentInputs/isPayment}} is set
         automatically.
 
     :  <dfn>locale</dfn> member
```

Comment:
(I dislike how GitHub doesn't let me tag comments to lines outside of the git diff, even if you expand the file...)

Do we need to do any validation of `browserBoundPubKeyCredParams` from the SecurePaymentConfirmationRequest dictionary during **4.1.8. Steps to validate payment method data** (line 749)? For example, what if the sequence is empty?

Reply:
Ack, thanks!